examples/rust/components/http-blobstore/wadm.yaml

```diff
@@ -20,7 +20,12 @@ spec:
       properties:
         # To use the locally compiled code in this folder, use the line below instead after running `wash build`:
         # image: file://./build/http_blobstore_s.wasm
-        image: ghcr.io/wasmcloud/components/http-blobstore-rust:0.2.0
+        image: ghcr.io/wasmcloud/http-blobstore-rust:0.3.0
+        id: http-blobstore
+        # config:
+        #   - name: http-blobstore-example
+        #     properties:
+        #       container_names: "ying,yang"
```

Comment:
nit: if this configuration is unused we can remove it from the YAML

Reply:
Removed; I had left them to show users how to pass container names.